region/crateria/west.json

```diff
@@ -892,7 +914,7 @@
                           {"ammo": [
                             {
                               "type": "PowerBomb",
-                              "count": 3
+                              "count": 2
```

Comment:
Where are you placing your bombs that you can make it through the entire room using only 3?

Reply:
Oh. It's because `canUsePowerBombs` already uses 1? That's gonna be super confusing.

Reply:
I think we probably oughta avoid the helpers when spending more than one PB.